src/components/Greetings.js

```diff
@@ -0,0 +1,18 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { randomGreeting } from '../redux/greeting';
+
+const Greetings = () => {
+  const greetings = useSelector((state) => state.greeting);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(randomGreeting());
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
The linters are suppose to help guide you to produce the best and standard code practices, kindly do not disable them.